### PR TITLE
Change text for after school to 'School/Organization Name'

### PR DESCRIPTION
**Pull request by ewjordan**
Text change for after school teachers.

Also removes an unused error div that was affecting layout (right edge of school info box was padded).

### CODE REVIEW

**Comment by ewjordan** [comment]
@poorvasingal and @tanyaparker, I'm not sure whether we want the (optional) on this or not. We don't include it for other school types, though we *do* when we're not using the school dropdown.
screen shot 2017-03-07 at 7 20 28 pm

screen shot 2017-03-07 at 7 57 17 pm

File: dashboard/app/views/devise/registrations/new.html.haml [hunk 1]
```diff
@@ -71,7 +71,6 @@
 
         #schooldropdown-block.itemblock{style: "display: none;"}
           = render partial: 'shared/school_info_registration', locals: {form_name: "user[school_info_attributes]", existing_school_info: nil, suppress_scrolling: true}
-          .error_in_field
```
Comment (breville) [inline dashboard/app/views/devise/registrations/new.html.haml:74]: this is never used to show an error?

Comment (ewjordan) [reply dashboard/app/views/devise/registrations/new.html.haml:74]: Nope, errors need to be connected to the individual fields they relate to, which are deeper inside the partial.

Comment (breville) [reply dashboard/app/views/devise/registrations/new.html.haml:74]: ah okay.  I think at one point we possibly had a single error for the entire dropdown.

**Comment by breville** [comment]
circleci failed?  unfortunately it's not loading for me to see what happened.

**Comment by ewjordan** [comment]
Yes, CircleCI is failing, for reasons that I think are unrelated to this PR, but I'll hold off on merging until it's resolved.

Failure is:

TextToSpeechTest
  test_sanitize_html                                              PASS (0.03s)
  test_tts_instructions_audio_file                                FAIL (0.04s)
Minitest::Assertion:         --- expected
        +++ actual
        @@ -1 +1 @@
        -"sharon22k/180/100/d41d8cd98f00b204e9800998ecf8427e/.mp3"
        +"sharon22k/180/100/256f8062d43018950cc245de47edf8b8/.mp3"
        test/models/concerns/text_to_speech_test.rb:47:in `block in <class:TextToSpeechTest>'
        test/testing/setup_all_and_teardown_all.rb:22:in `run'

  test_tts_instructions_text                                      FAIL (0.04s)
Minitest::Assertion:         Expected: ""
          Actual: "regular instructions"
        test/models/concerns/text_to_speech_test.rb:35:in `block in <class:TextToSpeechTest>'
        test/testing/setup_all_and_teardown_all.rb:22:in `run'

**Comment by tanyaparker** [comment]
If School Information says (Optional) I feel like that applies for all the fields below it so if that's true then I don't think those two fields need (optional) as well.

**Comment by tanyaparker** [comment]
What's the final screenshot for this? Main PR comment didn't finish loading the image.

@poorvasingal I find it weird that we have School X for some and School/Organization for others. And School Country but just State. Also I would think the order would be more like Country > State > Zip Code > Type > Name (with geographical fields grouped together) or at least Country > State > Type > Name > Zip Code.

File: dashboard/app/views/devise/registrations/new.html.haml (hunk 1 above)
Comment (ewjordan) [reply dashboard/app/views/devise/registrations/new.html.haml:74]: Yes, in the workshop registration version of this, there's a single error field, for some reason that wasn't working right for me here, so I changed it a bit. In the React rewrite things will be a little simpler, I hope.

**Comment by ewjordan** [comment]
Sorry about the screenshot, must not have uploaded right. Here's another:

screen shot 2017-03-08 at 3 20 44 pm

**Comment by ewjordan** [comment]
Final screenshot:

screen shot 2017-03-08 at 3 27 25 pm

**Comment by breville** [comment]
From a purely aesthetic point of view, worth making all the fields a bit shorter so that we don't need to wrap "Name" and "Zip Code"?

**Comment by ewjordan** [comment]
If I go to about 210px on the label, the line break goes away:

screen shot 2017-03-08 at 3 47 10 pm

I'm fine with that if everyone is.

**Comment by breville** [comment]
I like reducing the wrapping a lot.  Such a small offset versus the fields above, though, makes it feel as though there's a tiny misalignment.  I'd indent by a more dramatic amount, or bump the ones above by this small amount.  That way all the fields are either completely aligned or deliberately indented compared to each others :)

**Comment by ewjordan** [comment]
D'oh...what's happening above is that *all* of the title fields (including those not in the "School Information" box) share a CSS class, so when I changed it for the bottom ones the top ones were affected, too.  I'll try breaking them into two different classes, seeing if a bigger difference makes it look better.

**Comment by ewjordan** [comment]
Latest:

screen shot 2017-03-08 at 7 33 58 pm

**Comment by breville** [comment]
Ah, that looks good.  Though it's interesting to consider making the label fields wider on the upper set too, reducing some of the word wrapping in the labels.  That said, we could consider doing that separately.

**Comment by ashercodeorg** [comment]
This was not ready for merging, it appears there were still hard-coded test values in the PR (for example, in `analysisEvents.js`).

**Pull request merged**